config/rbac/kustomization.yaml

```diff
@@ -1,21 +1,21 @@
 resources:
-# All RBAC will be applied under this service account in
-# the deployment namespace. You may comment out this resource
-# if your manager will use a service account that exists at
-# runtime. Be sure to update RoleBinding and ClusterRoleBinding
-# subjects if changing service account names.
-- service_account.yaml
-- role.yaml
-- role_binding.yaml
-- leader_election_role.yaml
-- leader_election_role_binding.yaml
-# Comment the following 4 lines if you want to disable
-# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
-# which protects your /metrics endpoint.
-# - auth_proxy_service.yaml
-# - auth_proxy_role.yaml
-# - auth_proxy_role_binding.yaml
-# - auth_proxy_client_clusterrole.yaml
+  # All RBAC will be applied under this service account in
+  # the deployment namespace. You may comment out this resource
+  # if your manager will use a service account that exists at
+  # runtime. Be sure to update RoleBinding and ClusterRoleBinding
+  # subjects if changing service account names.
+  - service_account.yaml
+  - role.yaml
+  - role_binding.yaml
+  - leader_election_role.yaml
+  - leader_election_role_binding.yaml
+  # Comment the following 4 lines if you want to disable
+  # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
+  # which protects your /metrics endpoint.
+  # - auth_proxy_service.yaml
+  # - auth_proxy_role.yaml
+  # - auth_proxy_role_binding.yaml
+  - auth_proxy_client_clusterrole.yaml
```

Comment:
I was unsure about this - on main we currently have this role included in the bundle and have done for a long time but perhaps this is a remnant from the past? https://github.com/Kuadrant/kuadrant-operator/blob/main/bundle/manifests/kuadrant-operator-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml

Reply:
mhmm, this was probably a leftover when the kube-rbac-proxy sidecar was removed. Maybe we could just comment it and remove the file..we might need to assert auth capabilities are working tho

Reply:
I suspected the same to be honest - done, I've removed this